moped-editor/src/components/GridTable/GridTable.js

```diff
@@ -542,12 +547,14 @@ const GridTable = ({ title, query }) => {
                 </TableContainer>
 
                 {/*Pagination Management*/}
-                <GridTablePagination
-                  query={query}
-                  data={data}
-                  pagination={pagination}
-                  setPagination={setPagination}
-                />
+                {query.config.showPagination && (
+                  <GridTablePagination
+                    query={query}
+                    data={data}
+                    pagination={pagination}
+                    setPagination={setPagination}
+                  />
+                )}
```

Comment:
The showPagination setting was provided but not implemented, so I just added a few extra lines of code to take care of that. Whenever showPagination is false, it disables pagination entirely and shows everything in a single page.